### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Build Stage:
```
Comment (xansec) [inline Dockerfile:1]: Can we add a package stage here?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline Mayhemfile:1]: Multiple commands are not supported unless you are fuzzing both instrumented and uninstrumented calls of the same binary. I would either remove the extra commands or split into multiple Mayhemfiles. Be sure to also genericize the project name and omit the image.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I've made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**